packages/cerebral-website-tutorial/parts/05_update_state/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Controller } from 'cerebral'
+import App from './components/App'
+import { Container } from 'cerebral/react'
+import Devtools from 'cerebral/devtools'
+import { set } from 'cerebral/operators'
```

Comment:
yeah i'll replace all of them. thats why i asked for a npm release which has it included
if not planned soon i'll eject the tutorial app and link cerebral manually in the webpack conf (i did that already before)